index.html

```diff
@@ -35,6 +36,7 @@ <h2>Contact</h2>
 
 
    <div id="products">
+
    <h2 class="pro">Products</h2>
    <p class="description">Products we can offer:</p>
```

Comment:
It will be better using "section" tag instead of "div" tag to include the products tags to make it more readable and human friendly that is why we recommend using Semantic Html you can learn more about it following this link

[https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md]

Happy Coding!

![ClipWindowsGIF](https://user-images.githubusercontent.com/26763433/187165194-f6c3f272-74ac-4acb-934e-215d5549558d.gif)